gradle.properties

```diff
@@ -17,7 +17,6 @@ fabric_yarn_version=1.21.1+build.3:v2
 enabled_platforms=fabric,neoforge
 
 archives_base_name=minecraft-access
-mod_version=1.8.0+1.21.1
```

Comment:
This property is used in:
https://github.com/minecraft-access/minecraft-access/blob/7820fae574d450515181e8c1fde96a9c7306c63c/build.gradle#L37